FutureNNAimbot/Program.cs

```diff
@@ -319,19 +360,19 @@ static void Main(string[] args)
                         Process.GetProcessesByName(game)[0].Kill();
                         if (File.Exists($"trainfiles/{game}.weights"))
                         {
-                            File.Copy($"trainfiles/{game}.weights", $"darknet/{game}.weights",true);
+                            File.Copy($"trainfiles/{game}.weights", $"darknet/{game}.weights", true);
                             Process.Start("cmd", @"/C cd " + Application.StartupPath + $"/darknet/ & {game}_trainmore.cmd");
                         }
                         else Process.Start("cmd", @"/C cd " + Application.StartupPath + $"/darknet/ & {game}.cmd");
 
-                        Console.WriteLine("Then you finished training write \"done\" in this console!");
+                        Console.WriteLine("When you finished training, write \"done\" in this console.");
```

Comment:
When you have finished training the aimbot,

Reply:
Shouldn't it be NN instead of aimbot?